contents/docs/deployment/securing-posthog.md

```diff
@@ -31,7 +31,19 @@ TRUST_ALL_PROXIES=True
 
 > If you're on Heroku, you are behind a proxy by default, so you'll need to add `IS_BEHIND_PROXY=True`. Heroku automatically overrides `X-Forwarded-For`, so you can use `TRUST_ALL_PROXIES=True`.
 
+## Secure Cookies
 
+Starting with PostHog 1.13.0, we introduced a `SECURE_COOKIES` flag. This defaults to "False" when PostHog is running on `DEBUG` or `TEST` mode (generally the case when running locally) and "True" in production (when those modes are not on).
+
+While this defaults to "True" in environments that are not `TEST` or `DEBUG`, you may need to toggle this off for setup or testing purposes in a production instance. However, remember that **secure cookies should always be on when handling live data (i.e. in production).** This flag affects cookies used in Django sessions, CSRF tokens, and Toolbar login. In short, it ensures the security of your users' data, hence it is so important.
```

Comment:
"In short, it ensures the security of your users' data, hence it is so important." <-- as a site owner, who are "my users" is not clear. 

"Turning this off makes my site less secure?"... is what might be asked and that will not be true.

It will make your local posthog installation less secure and all the users who are there. But that's not "your users" if you're the owner of a popular website. "Security of your posthog instance" could be a more accurate, yet comparably serious statement.

Reply:
I believe an insecure instance affects everyone down to the end-user, which is why I made the statement. I get your point though. Updated it!